Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -78,6 +78,19 @@
       - Screwing
     useSound:
       collection: Screwdriver
+  - type: MultipleTool
+    statusShowBehavior: true
+    entries:
+      - behavior: Screwing
+        useSound:
+          collection: Screwdriver
+      - behavior: Prying
+        speedModifier: 0.4
+        useSound:
+          path: /Audio/Items/crowbar.ogg
```

Comment:
This does not actually alter the speed of prying stuff like doors. The Prying component has its own modifier

Reply:
I tested it and it does seem to affect the speed of prying doors. Before I added it, screwdrivers and crowbar pried doors at the same speed.

Reply:
Oh shoot im so sorry. I didnt even push the change that adds the speed modifier to the actual PryingComponent.